avocado/core/job.py

```diff
@@ -392,6 +394,14 @@ def soft_abort(msg):
             if os.path.exists(proc_latest):
                 os.unlink(proc_latest)
 
+    @property
+    def size(self):
+        """Job size is the sum of all test suites sizes."""
+        size = 0
```

Comment:
Given that a test suite size is guaranteed to exist and return an integer, this could simply be:

```python
return sum([suite.size for suite in self.test_suites])
```

Reply:
Sure, it was on my todo to simplify this, but forgot. Nice catch.